docs/api/js/features.md

```diff
@@ -61,13 +61,13 @@ Use custom signature `0x123456` for account, while using paymaster with address
 
 ## Encoding paymaster params
 
-While the paymaster feature by itself does not impose any limitations on values of the `paymasterInput`, the Matter Labs team endorses certain types of [paymaster flows](../../dev/zksync-v2/aa.md#built-in-paymaster-flows) that are processable by EOAs.
+While the paymaster feature by itself does not impose any limitations on values of the `paymasterInput`, the Matter Labs team endorses certain types of [paymaster flows](../dev/../../dev/developer-guides/transactions/aa.md#built-in-paymaster-flows) that are processable by EOAs.
```

Comment:
Same here. The `..` is used to get up in the folder tree, you don't need to go up in the tree, down and then up again